### DISCUSSION TITLE
Can we get an ETA on the next release please?

### DISCUSSION THREAD

**Discussion by johnsonlin** [jscutlery/semver#714] [Q&A] [answered]
We really want to use the `commitParserOptions`🙏

**Comment by edbzn** [comment]
Hi, our pipeline is broken due to an issue with Nx, it was reported to the team, and we are waiting for a fix.

**Comment by edbzn** [comment, marked as answer]
Released in version [3.1.0](https://github.com/jscutlery/semver/releases/tag/semver-3.1.0).
